app/controllers/posts_controller.rb

```diff
@@ -5,6 +5,29 @@ def index
   end
 
   def show
+    # @user = User.find_by(id: params[:user_id])
     @post = Post.where(author_id: params[:user_id], id: params[:id]).first
+    # @current_user = current_user
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧